tests/bagatom/test_bagToXML.py

```diff
@@ -0,0 +1,208 @@
+    )
+    assert len(fileCount) == 1
+    assert fileCount[0].text == str(FILE_COUNT)
+
+
+def test_return_tree_has_payloadSize(bagxml):
+    """
+    Verify the return value has a payloadSize element.
+    """
+    payloadSize = bagxml[0].xpath(
+        '/a:codaXML/a:payloadSize',
+        namespaces={'a': bagatom.BAG_NAMESPACE}
+    )
+    assert len(payloadSize) == 1
+    assert payloadSize[0].text == str(PAYLOAD_SIZE)
+
+
+def test_return_tree_has_bagitversion(bagxml):
+    """
+    Verify the return value has a bagitVersion element.
+    """
+    bagitVersion = bagxml[0].xpath(
+        '/a:codaXML/a:bagitVersion',
+        namespaces={'a': bagatom.BAG_NAMESPACE}
+    )
+    assert len(bagitVersion) == 1
+    assert bagitVersion[0].text == '0.96'
+
+
+def test_return_tree_has_lastVerified(bagxml):
+    """
+    Verify the return value has a lastVerified element.
+    """
+    lastVerified = bagxml[0].xpath(
+        '/a:codaXML/a:lastVerified',
+        namespaces={'a': bagatom.BAG_NAMESPACE}
+    )
+    assert len(lastVerified) == 1
+    assert lastVerified[0].text is None
+
+
+def test_return_tree_has_lastStatus(bagxml):
+    """
+    Verify the return value has a lastStatus element.
+    """
+    lastStatus = bagxml[0].xpath(
+        '/a:codaXML/a:lastStatus',
+        namespaces={'a': bagatom.BAG_NAMESPACE}
+    )
+    assert len(lastStatus) == 1
+    assert lastStatus[0].text is None
+
+
+def test_return_tree_has_bagInfo(bagxml):
+    """
+    Verify the return value has a bagInfo element.
+    """
+    bagInfo = bagxml[0].xpath(
+        '/a:codaXML/a:bagInfo',
+        namespaces={'a': bagatom.BAG_NAMESPACE}
+    )
+    assert len(bagInfo) == 1
+    assert len(bagInfo[0].getchildren()) == 1
+
+
+def test_return_tree_check_item_children(bagxml_with_tags):
+    """
+    Verify the return value has the correct item child elements
+    and values.
+    """
+    # Get all the item values and put them into a list.
+    items = bagxml_with_tags[0].xpath(
+        '/a:codaXML/a:bagInfo/a:item//text()',
+        namespaces={'a': bagatom.BAG_NAMESPACE}
+    )
+
+    assert 'Bag-Size' in items
+    assert '1GB' in items
+    assert 'Payload-Oxum' in items
+    assert '1500.3' in items
+    assert 'Source-Organization' in items
+    assert 'Test Org' in items
+
+
+def test_return_tree_has_no_bagging_date(bagxml):
+    """
+    Verify the return value does not have a baggingDate element.
+    """
+    baggingDate = bagxml[0].xpath(
+        '/a:codaXML/a:baggingDate',
+        namespaces={'a': bagatom.BAG_NAMESPACE}
+    )
+    assert len(baggingDate) == 0
+
+
+def test_return_tree_with_bagging_date(bagxml_with_bagging_date):
+    """
+    Check that the return tree baggingDate element value is correct.
+    """
+    baggingDate = bagxml_with_bagging_date[0].xpath(
+        '/a:codaXML/a:baggingDate',
+        namespaces={'a': bagatom.BAG_NAMESPACE}
+    )
+    assert baggingDate[0].text == BAGGING_DATE
```

Comment:
It seems these last two tests break the consistency in test names that your other tests had. By that I mean that it seems that these tests should be called '...no_baggingDate' and '...with_baggingDate', respectively.